app/experimenter/targeting/constants.py

```diff
@@ -142,6 +142,33 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEW_PROFILE_HAS_PIN_NEED_DEFAULT_WINDOWS_1903 = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default w/ pin"
+    ),
+    slug="first_run_new_profile_need_default_has_pin",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default w/ pin"
+    ),
+    targeting=(
+        "{first_run} && {has_pin}".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting,
+            has_pin=HAS_PIN,
+        )
+    ),
+    desktop_telemetry=(
+        "{first_run} AND {has_pin}".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.desktop_telemetry,
+            has_pin=HAS_PIN,
+        )
+    ).format(first_run=FIRST_RUN.desktop_telemetry, has_pin=HAS_PIN),
```

Comment:
@jaredlockhart not sure about two formats (line 162 and line 166) syntax for desktop_telemetry. Does that look right?

Reply:
Oh, hm I based it off `FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_MOTION` targeting, but that is the only one that has two formats. 🤔 I can remove it since it  `FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903` has the targeting I need.

Reply:
Oh wow I completely missed that.  Yeah that won't work like that, the second call is a no-op since the format strings will already be replaced.  You only need to call format once.

Reply:
Thanks for clarifying, I updated it! Should I also update the existing targeting that uses another format for telemetry on line 139 for `FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_MOTION ` or should that be a separate pr?

```
    desktop_telemetry=(
        "{first_run} AND !userPrefersReducedMotion".format(
            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.desktop_telemetry,
        )
    ).format(first_run=FIRST_RUN.desktop_telemetry),
    ```

Reply:
My understanding we discourage changing existing advanced targeting,  if needed we can remove and add new one when desktop_telemetry feature is available in auto analysis.

Reply:
I would say in general, adding a new targeting is preferred.  Modifying or removing existing ones can definitely be done and won't break anything, but it might interrupt someone elses planning, so if you can get away with just adding a new one that's generally a better idea.